### ISSUE TITLE
This issue is fixed by latest clidriver which packages correct libstdc++.0.6.dylib library. Thanks.

### ISSUE THREAD

**Issue by a473000** [ibmdb/python-ibmdb#802]
can any one help I get the same problem and I am a not as fluent  yet in sorting it out so please plain english how to sort it out Thank you I have assignment to do in db2 but I can't.

**Comment by Earammak** [comment]
@a473000 Can you please share the detailed explanation about the issue you are facing here.

**Comment by a473000** [comment]
Hi thank you for your time , so i am using anaconda and Jnotbook or labs, on mac mini m1 and when I do import ibm_db i get error like the one on top of this post, exactly, I did install brew and other things but I do not know what they     do.
Next on macbook pro intel older osx fresh anaconda and then terminal pip install ibm_db 
Then Jupiter note book import ibm_db and the same error. 
I did follow all the. Steps from coursera and all the things we working now we do sql and I can't get it the tool to do it. Any help will be appreciated thank you
